packages/web-shared/components/FeatureFeed/Feed.js

```diff
@@ -52,12 +37,9 @@ const Feed = (props) => {
     );
   }
 
-  const features = props?.data?.features?.filter(
-    (feature) => feature.cards !== null
-  );
-  const renderedFeatures = features?.map(renderFeature).filter(isNil);
+  const renderedFeatures = compact(filter(data.features, feature => !feature?.cards?.length));
```

Comment:
I would like someone to double check this line specifically. Paranoid :) . I read the original line as "toss features that have cards". Is that correct.

Reply:
Yeah, I agree the code is hard to follow.

I think the original `const features` code was filtering features that didn't have any "cards". Then the `const renderedFeatures` was filtering out "features" that were empty/nil.

There are many different possibilities to why features might have cards or might be returning null. One example is If you are logged in, some features show up based on your campus. Like a Continue Watching, or For You.

It also looks like this code is breaking the feed. I'm only seeing one feature show up using `data-church="liquid_church"` and `data-feature-feed="FeatureFeed:d292fd8c-5b9f-4cd7-a2fe-9d7d0914c34f"`

after
<img width="575" alt="image" src="https://github.com/ApollosProject/apollos-embeds/assets/2528817/c2a925ad-ce46-4f09-86d0-7f723fcf0b32">

before
![CleanShot 2023-11-03 at 11 53 21@2x](https://github.com/ApollosProject/apollos-embeds/assets/2528817/ca37886b-cc6b-41f1-ad85-0ec240a9f330)